easybuild/easyconfigs/i/intel-compilers/intel-compilers-2021.1.2.eb

```diff
@@ -0,0 +1,31 @@
+name = 'intel-compilers'
+version = '2021.1.2'
+
+homepage = 'https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html'
+description = "Intel C, C++ & Fortran compilers (classic and oneAPI)"
+
+toolchain = SYSTEM
+
+# see https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit/download.html
```

Comment:
I checked this out and can't find these download URLs from there...have you got a secret stash?

Reply:
Is it also true going forward that we will have separate download URLs for the compilers? Are they guaranteed to be released as a pair?

Reply:
This is probably a better URL: https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html 

I have no idea what things are going to be like going forward, but if Intel gets creative with versioning or releases in the future, we can deal with it then?

I would expect them to always issue a new release for the Intel C++ and Fortran compilers combined, but who knows...

Reply:
Ok